pkg/services/org/orgtest/fake.go

```diff
@@ -23,7 +23,7 @@ type FakeOrgService struct {
 }
 
 func NewOrgServiceFake() *FakeOrgService {
-	return &FakeOrgService{}
+	return &FakeOrgService{ExpectedOrg: &org.Org{ID: 4}}
```

Comment:
Shouldn't this be 0?

```suggestion
	return &FakeOrgService{ExpectedOrg: &org.Org{ID: 0}}
```

Reply:
@andresmgot No. This Org is tested against in https://github.com/grafana/grafana/pull/76937/files#diff-8f8b2e7e8794fdda540742ab01df72e23bcb091b6bbe05a0ec33f98888d2c9a1R119.

This `FakeOrg` struct can be changed to an in-memory implementation. This would improve tests. I can do this (maybe in another PR?) if you want.